rocketpy/tools.py

```diff
@@ -232,32 +232,39 @@ def bilinear_interpolation(x, y, x1, x2, y1, y2, z11, z12, z21, z22):
     ) / ((x2 - x1) * (y2 - y1))
 
 
-def get_distribution(distribution_function_name):
+def get_distribution(distribution_function_name, rng_generator=None):
     """Sets the distribution function to be used in the monte carlo analysis.
 
     Parameters
     ----------
     distribution_function_name : string
         The type of distribution to be used in the analysis. It can be
         'uniform', 'normal', 'lognormal', etc.
+    rng_generator : np.random.Generator, optional
+        The random number generator to be used. If None, the default generator
+        is used.
```

Comment:
Let it cleaerer that the default generator is the numpy default gerator.

Reply:
Made it clearer on [d18408e](https://github.com/RocketPy-Team/RocketPy/pull/649/commits/d18408eb0679f06f89b42a2999b6a3d8c5c5698f).